docs/source/daml/intro/7_Composing.rst

```diff
@@ -211,18 +211,28 @@ DAML's execution model
 
 DAML's execution model is fairly easy to understand, but has some important consequences. You can imagine the life of a transaction as follows:
 
-1. A party submits a list of ``Commands``.
-2. The transaction is interpreted, meaning the ``Update`` corresponding to each command is evaluated in the context of the ledger to calculate all consequences, including transitive ones (consequences of consequences, etc.).
-3. The views of the transaction that parties get to see (see :ref:`privacy`) are calculated in a process called *blinding*, or *projecting*.
-4. The blinded views are distributed to the parties.
-5. The transaction is *validated* based on the blinded views and a consensus protocol depending on the underlying infrastructure.
-6. If validation succeeds, the transaction is *committed*.
+Command
+  A user submits a list of Commands via the Ledger API of a Participant Node, acting as a `Party` hosted on that Node. That party is called the requester.
```

Comment:
Side node: We should probably decide on whether we want to call this requester or committer. Currently the ledger model uses requester but our code, DAML studio and other places use committer.

Reply:
submitter and committer have different meanings elsewhere:
- Submitter being the submitting participant
- Committer being special nodes in some topologies
I've therefore gone all in on requester.

Reply:
Then we seem to use `committer` wrong in quite a lot of cases. E.g., there is an error message in DAML Studio that something is not visible to the committer which refers to the party submiting the commands.